roles/keycloak/tasks/start_keycloak.yml

```diff
@@ -11,5 +11,5 @@
     url: "{{ keycloak.health_url }}"
   register: keycloak_status
   until: keycloak_status.status == 200
-  retries: 25
-  delay: 10
\ No newline at end of file
+  retries: "{{ keycloak_service_start_retries }}"
+  delay: "{{ keycloak_service_start_delay }}"
```

Comment:
@guidograzioli I suspect users will need to tweak that, if nothing else, because of slow (or very fast system)